src/ios/LottieReactNative/ContainerView.swift

```diff
@@ -82,14 +82,26 @@ class ContainerView: RCTView {
         applyProperties()
     }
 
-    func play(fromFrame: AnimationFrameTime? = nil, toFrame: AnimationFrameTime, completion: LottieCompletionBlock? = nil) {
+    func play(fromFrame: AnimationFrameTime? = nil, toFrame: AnimationFrameTime) {
+        let callback: LottieCompletionBlock = { animationFinished in
+            if let onFinish = view.onAnimationFinish {
```

Comment:
`view.onAnimationFinish` should be `self.onAnimationFinish`. Same as the one in `play()`

Reply:
Yep, forgot to move that over. Great catch!